project.js

```diff
@@ -0,0 +1,52 @@
+(()=>{
+    let index = 0;
+    
+    const peronName = document.getElementById('name');
+    const personHeight=document.getElementById('height');
+    const personGender=document.getElementById('gender');
+    const personSkin_color=document.getElementById('skin');
+    const nextPersonBtn = document.getElementById('btn2');
+
+    const prevPersonBtn=document.getElementById('btn1');
+    
+    const swapiPeopleAPI = (index = 1) => `https://swapi.co/api/people/${index}/?format=json`;
+    
+    function getPerson(index = 1) {
+      return fetch(swapiPeopleAPI(index)).then(data => data.json());
+    }
+    
+    async function updateNextPerson() {
+      const {name} = await getPerson(++index);
```

Comment:
you need to call get person only once, u can Destructure  properties on the same like you have done for name